subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -544,6 +551,23 @@ public AbstractTestTask setTestNameIncludePatterns(List<String> testNamePattern)
         return this;
     }
 
+    @Option(option = "fail-fast", description = "Stops test execution after the first failed test")
```

Comment:
Using the description has a dot at the end of the sentence (see other usages of `@Option`).

Reply:
done